### ISSUE TITLE
Magnificent Flying Carpet mount not showing up on my other characters on the same account

### ISSUE THREAD

**Issue by DonitoFlakes** [gamefreedomgit/Maelstrom#5789]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
My Character Alianaa Has Tailoring to skill 450 and has the mount Magnificent Flying Carpet she can use normally but my Other characters on the same account cannot see the Magnificent Flying carpet on the mount menu Tab so they cannot use it Namely Eliannaa I was advised by a GM to report this as a bug
**How to reproduce:**

**How it should work:**
All other characters on the same account should be able to see the mount on the mount tab and be able to use it once they are of the correct character level since one character already has the tailoring skill to the appropriate level and mounts including profession mounts are account wide.

**Database links:**
![GM response](https://github.com/gamefreedomgit/Maelstrom/assets/163668771/551009a2-1eb4-4554-a027-1f6a227164ba)

![1](https://github.com/gamefreedomgit/Maelstrom/assets/163668771/e77e26c8-8eae-4758-8c19-3ba6e7ea2f63)
![2](https://github.com/gamefreedomgit/Maelstrom/assets/163668771/03537cde-c5a1-4a37-9219-498faab72e3c)
![3](https://github.com/gamefreedomgit/Maelstrom/assets/163668771/a4db17c9-d44a-45d5-ad4a-52596629591c)
![4](https://github.com/gamefreedomgit/Maelstrom/assets/163668771/328c4a23-8871-4957-88a5-86d72b42b02c)
![5](https://github.com/gamefreedomgit/Maelstrom/assets/163668771/8557c382-6b47-44ab-9388-731c4603cbe6)
![6](https://github.com/gamefreedomgit/Maelstrom/assets/163668771/861eea3f-0936-4606-a8ff-ad21361aa449)

**Comment by UltraNix** [comment]
This mount is available only to tailors.
```
mounts including profession mounts are account wide.
```
Mounts are account-wide, but not professions. Do you have tailoring profession on both characters?

**Comment by DonitoFlakes** [comment]
I only have it on one of my characters, that is why i double checked with
GM Neon Via Support ticket and confirmed that the intended purpose was if
one of the characters on the same account has the profession for it, it
would still be usable i can send the screenshots of the full conversation
if that would help.

**Comment by DonitoFlakes** [comment]
Also to add for the sake of it as shown in the photos the Engineering
mounts are usable on both characters even though only one of them has the
engineering profession and GM Neon stated that that was the intended
result. So honestly I’m confused because of conflicting information I’ve
been given.

**Comment by UltraNix** [comment]
@DonitoFlakes What is your riding skill on second char?

**Comment by UltraNix** [comment]
Yep - you are right. There is something wrong with loading such mounts. Will be fixed.

**Comment by DonitoFlakes** [comment]
Hello so just to be clear since I also want to confirm for the future, the fact that profession mounts are able to be used by characters that don't have the specific profession is a bug? and as per my prior example [Turbocharged Flying Machine] should not be available to my other characters unless they have Engineering as well is that correct?

**Comment by UltraNix** [comment]
Professions are not required for account-wide mounts.
You can use it on char with no skill in engineering.

**Comment by DonitoFlakes** [comment]
Both of my characters have Master Riding, Cold weather flying and Flight
master's License

**Comment by UltraNix** [comment]
Fixed.
